Examples/Algorithms/TrackFinding/src/SpacePointMaker.cpp

```diff
@@ -102,75 +107,18 @@ ActsExamples::ProcessCode ActsExamples::SpacePointMaker::execute(
 
     for (auto [moduleGeoId, moduleSourceLinks] : groupedByModule) {
       // find corresponding surface
-      const Acts::Surface* surface =
```

Comment:
Where did the code from the lambda go?

Reply:
Thanks for the comments, Andi. What is the code from the lambda?